### PR TITLE
Add KDE4 Dataset

### PR DESCRIPTION
**Pull request by abhishekkrthakur**

### CODE REVIEW

File: datasets/kde4/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+### Data Instances
+
+Here are some examples of questions and facts:
+
```
Comment (lhoestq) [inline datasets/kde4/README.md]:
```suggestion
[More Information Needed]
```

File: datasets/kde4/kde4.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+92 languages, 4,099 bitexts
+total number of files: 75,535
+total number of tokens: 60.75M
+total number of sentence fragments: 8.89M
+"""
+_HOMEPAGE_URL = "http://opus.nlpl.eu/KDE4.php"
+_CITATION = """\
+@InProceedings{TIEDEMANN12.463,
+  author = {J�rg Tiedemann},
```
Comment (lhoestq) [inline datasets/kde4/kde4.py:33]:
```suggestion
  author = {J{\"o}rg Tiedemann},
```

File: datasets/kde4/README.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
+---
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- fi
+- nl
```
Comment (lhoestq) [inline datasets/kde4/README.md:8]: some languages are missing no ?

**Review by lhoestq — COMMENTED** [review]
Crazy how 1/3 of the PRs I'm doing are yours xD

left a few comments :

**Comment by abhishekkrthakur** [comment]
@lhoestq fixed :)

**Review by lhoestq — APPROVED** [review]
Thanks !

**Pull request merged**